wp-content/plugins/core/src/Assets/Editor_Assets_Enqueuer.php

```diff
@@ -11,7 +11,13 @@ class Editor_Assets_Enqueuer extends Assets_Enqueuer {
 	public function register(): void {
 		$args = $this->get_asset_file_args( $this->assets_path . self::ASSETS_FILE );
 
-		add_editor_style( $this->assets_path_uri . self::EDITOR_FILE_NAME . '.css', );
+		wp_enqueue_style(
```

Comment:
I think you need to add an `is_admin()` check at the top of this function and bail early if the current view is not the admin.

Reply:
Ah good call @dpellenwood!  this one is updated, tested, and fixed now.